### PR TITLE
Added in a utilisation test

### PR DESCRIPTION
**Pull request by SimonBiggs**
Hi @pchlap and @rnfinnegan,

I wanted to reproduce the error over at https://github.com/pyplati/platipy/issues/121 within a pytest test. Here is that attempt. I wasn't able to reproduce the error locally however. So... up to you if you think this test is helpful or not :slightly_smiling_face:.

Cheers,
Simon

### CODE REVIEW

File: platipy/imaging/visualisation/tests/test_use_case.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+            output_dir=patient_dir,
+            spacing=None,
+        )
+
+        structure_paths = patient_dir.glob(f"{prefix}*.nii.gz")
+
+        contours = {}
+        for path in structure_paths:
+            filename = _name_with_all_suffixes_removed(path)
+            contour_name = filename.removeprefix(prefix)
```
Comment (pchlap) [inline platipy/imaging/visualisation/tests/test_use_case.py:64]:
```suggestion
            contour_name = filename.replace(prefix, "")
```

Comment (pchlap) [reply platipy/imaging/visualisation/tests/test_use_case.py:64]: Test is failing here, not sure what `removeprefix` is part of?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -11,6 +11,15 @@ request](https://github.com/pyplati/platipy/compare) to merge your changes into
 (**master**). At this point the automated tests will run and maintainers will review your
 submission before merging.
 
+## Example Installation
```
Comment (pchlap) [inline CONTRIBUTING.md:14]: Thanks @SimonBiggs for adding this in!

**Review by pchlap — APPROVED** [review]
Thanks @SimonBiggs :) So happy to have our first contribution to platipy outside of Rob and me.

It seems like the issue ended up being a bug in the user's code? I would still like to merge this through. Useful to have an additional test and I appreciate your updates to the CONTRIBUTING doc :)

File: platipy/imaging/visualisation/tests/test_use_case.py (hunk 1 above)
Comment (SimonBiggs) [reply platipy/imaging/visualisation/tests/test_use_case.py:64]: Ahh, bumma, that's a Python 3.9 method added to the str object. It makes sure that it truly is a prefix (the suggested code your providing would be an issue if the prefix also existed in the middle of the string somewhere).

The `removeprefix` function looks like this:

```python
def removeprefix(self: str, prefix: str, /) -> str:
    if self.startswith(prefix):
        return self[len(prefix):]
    else:
        return self[:]
```
https://peps.python.org/pep-0616/#specification

File: CONTRIBUTING.md (hunk 2 above)
Comment (SimonBiggs) [reply CONTRIBUTING.md:14]: My pleasure :) The best way to grow documentation I think is to have people try and use it and then contribute where they found gaps :)

**Comment by SimonBiggs** [comment]
> It seems like the issue ended up being a bug in the user's code?

Yup, that's the case

> So happy to have our first contribution to platipy outside of Rob and me.

Glad to be able to be the first outside contributor :)

> Useful to have an additional test

Yeah, I thought there might be value to have it in there anyway.

**Comment by SimonBiggs** [comment]
Oops, my bad with that plt import. I need to move the matplotlib import to the top. I'm in Sydney this week. I'd be more than happy @pchlap if you jumped in and made that change as I won't be able to get to it until next week.

**Comment by pchlap** [comment]
> Oops, my bad with that plt import. I need to move the matplotlib import to the top. I'm in Sydney this week. I'd be more than happy @pchlap if you jumped in and made that change as I won't be able to get to it until next week.

Yep happy too. I'm always a bit unsure on how to best collaborate on a PR coming from a fork. Since I can't write to your fork... Do I just start a fresh PR?

**Comment by SimonBiggs** [comment]
Ideally, I'd set the following permission and let you edit the commits:

https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork

But, because I made the PR from an organisation account I don't believe I have the option to do that. What I'll do, is I'll directly give you commit access to my fork. Then you'll be free to just directly make changes.

**Comment by SimonBiggs** [comment]
Kk, I've given you commit access to my fork. Let me know if that lets you make changes.

**Comment by pchlap** [comment]
> Kk, I've given you commit access to my fork. Let me know if that lets you make changes.

Thanks @SimonBiggs, that works well :)

**Comment by pchlap** [comment]
Thanks a lot for this @SimonBiggs. I will merge it through.

**Pull request merged**